storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2785,9 +2792,8 @@ class Rdb_trx_info_aggregator : public Rdb_tx_list_walker {
       }
 
       std::string query_str;
-      LEX_STRING *const lex_str = thd_query_string(thd);
-      if (lex_str != nullptr && lex_str->str != nullptr) {
-        query_str = std::string(lex_str->str);
+      if (thd->query().str != nullptr) {
```

Comment:
This dropped the check "lex_str != nullptr", aka "thd->query() != nullptr" in 5.7. Maybe it can never be nullptr?